lib/keys.js

```diff
@@ -21,25 +21,34 @@ Keys.prototype.get = function(key, fn) {
 };
 
 /*!
- * Generate a key using cryptographically strong pseudo-random data.
- */
+* Generate a key using cryptographically strong pseudo-random data.
+*/
 
 Keys.prototype.generate = function() {
   return crypto.randomBytes(256).toString('hex');
 };
 
 /*!
- * Create a new key and save it in the keys file.
- */
+* Create a new key and save it in the keys file.
+*/
 
 Keys.prototype.create = function(fn) {
   var key = this.generate()
-    , keys = this;
+  , keys = this;
 
+  if (this.path.indexOf('.dpd') === 0 && !fs.existsSync('.dpd') ){
+    if( fs.existsSync('node_modules/deployd/.') ){
+      fs.mkdirSync('.dpd');
+    }else{
+      console.error("ERROR! You don't have deployd module in node_modules directory, won't create .dpd directory");
+    }
+  }
```

Comment:
The prompt sounds good, but see my comment below. I think that's the better place to put this user-land facing code.